src/cmd/flux-mini.py

```diff
@@ -225,6 +225,7 @@ class Xcmd:
         "bcc": "--bcc=",
         "log": "--log=",
         "log_stderr": "--log-stderr=",
+        "dependency": "--dependency=",
```

Comment:
I just realized that this could also be handled with the OpenMP-style schema without a special variable by using the `--define` argument of bulksubmit:
```
seq 1 5 | flux mini bulksubmit --define=prev="int(x)-1" --dependency=string:{}?type=out --dependency=string:{.prev}?type=in hostname
```

Reply:
Unfortunately, I think the OpenMP-style would have to be used for this case since jobids are not known at the time of submission in bulksubmit/submit `--cc`.